CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * Bug fixes:
     * fixed 483 by extracting a common CMR query method for collections and granules using SearchAfter header
     * Added VCR support for verifying the API call to CMR and the parsing of returned results without relying on CMR availability post development
+  * [#562](https://github.com/nsidc/earthaccess/issues/562): The destination
```

Comment:
Indentation is off by 2 spaces with the items above

Reply:
Oops! I'll sort that out with my other PR, where I've reformatted the section. I didn't want to create much of a merge conflict for myself here.